modules/trusted-firmware-m/Kconfig

```diff
@@ -233,6 +233,40 @@ config TFM_REGRESSION
 	  When enabled, this option signifies that the TF-M build includes
 	  the Secure and the Non-Secure regression tests.
 
+config TFM_PSA_TEST_CRYPTO
+	bool "Crypto tests"
+	help
+	  Enable the PSA Crypto test suite.
+
+config TFM_PSA_TEST_PROTECTED_STORAGE
+	bool "Storage tests"
+	help
+	  Enable the PSA Protected Storage test suite.
+
+config TFM_PSA_TEST_INTERNAL_TRUSTED_STORAGE
+	bool "Internal Trusted Storage tests"
+	help
+	  Enable the PSA Protected Storage test suite.
+
+config TFM_PSA_TEST_STORAGE
+	bool "Storage tests"
+	help
+	  Enable the PSA Storage test suite. This is a combination of the
+	  protected storage and internal trusted storage tests.
+
+config TFM_PSA_TEST_INITIAL_ATTESTATION
+	bool "Initial attestation tests"
+	help
+	  Enable the PSA Initial Attestation test suite.
+
+if TFM_PSA_TEST_CRYPTO || TFM_PSA_TEST_INITIAL_ATTESTATION
+config MAIN_STACK_SIZE
```

Comment:
I am not so happy with this. Why should the main thread be affected, by default? Perhaps we can do this in the test/sample we write that demonstrates the services? Or am i missing something?

Reply:
Added as depends on above instead.